### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by jamesmarlowe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+ngx_zmq
+=======
+An upstream ZeroMQ module for nginx
+
+Reference Materials
```
Comment (chaseholland) [inline README.md]: :+1:

File: README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+Start a zmq server (example included in us.cpp)
+```
+g++ us.cpp -lzmq
+./a.out
+```
+Curl nginx to get the response from the zmq server with
+```$ curl -X POST -d 'hello' http://localhost/zmq```
+
+Which will give
+`World`
```
Comment (chaseholland) [inline README.md]: No hello? :(

Comment (jamesmarlowe) [inline README.md]: curl with hello. Its close enough

File: restart.sh [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```
Comment (chaseholland) [inline restart.sh:1]: Remember we're open sourcing this, so you might want to include a blerb about why this exists in the README :)

Comment (jamesmarlowe) [inline restart.sh:1]: I did already https://github.com/neloe/ngx_zmq/pull/2/files#diff-04c6e90faac2675aa89e2176d2eec7d8R40

**Pull request merged**